src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -76,6 +78,37 @@ public void setStudyTrackerFilePath(Path studyTrackerFilePath) {
         userPrefs.setStudyTrackerFilePath(studyTrackerFilePath);
     }
 
+    @Override
+    public void addAlias(Alias aliasToAdd) {
+        List<Alias> aliases = userPrefs.getUserAliases();
+
+        // if alias already defined, replace command word with incoming aliasToAdd
+        aliases.removeIf(al -> al.userAlias.equals(aliasToAdd.getUserAlias()));
```

Comment:
not sure if this is fine, or if it should thrown an exception instead

Reply:
Appreciate the review, Joenz! 

1. Yes, it's intended for the user to alias invalid commands. It's meant to allow aliases of something like `edit r/5` (no spot/ is specified here), so that aliases can be extended even further. StudyTracker will throw appropriate errors when trying to execute any alias anyway.
2. Have switched this to throw an exception in UnaliasCommand::execute() to match functionality of DeleteCommand::execute(). Thanks for the catch!